pkgs/by-name/sq/sqlitestudio/package.nix

```diff
@@ -0,0 +1,81 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libsForQt5,
+  readline,
+  tcl,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+stdenv.mkDerivation rec {
+  pname = "sqlitestudio";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "pawelsalawa";
+    repo = pname;
+    rev = version;
+    hash = "sha256-5oBYv8WxyfVvvqr15XApvn6P/lBxR8b6E+2acRkvX0U=";
+  };
+
+  nativeBuildInputs =
+    with libsForQt5.qt5;
+    [
+      qmake
+      qttools
+      wrapQtAppsHook
+    ]
+    ++ [ copyDesktopItems ];
+
+  buildInputs =
+    with libsForQt5.qt5;
+    [
+      qtbase
+      qtsvg
+    ]
+    ++ [
+      readline
+      tcl
+    ];
+
+  qmakeFlags = [ "../../SQLiteStudio3" ];
```

Comment:
According to build instructions [here](https://github.com/pawelsalawa/sqlitestudio/wiki/Instructions_for_compilation_under_Linux#main-project) we need to run `qmake ../../SQLiteStudio3`.